### PR TITLE
feat(Readme): README now includes all information needed to create a new web component

### PR DESCRIPTION
**Pull request by jeromecambon**
- README now includes all information needed to create a new web component, from an existing one or from scratch.
- Note that this is a move from the UID SDK Readme, with many updates
- Minor fix for gen-widget script

Covers [UID-584](https://bonitasoft.atlassian.net/browse/UID-584)

### CODE REVIEW

File: components/uid-text/package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "test:watch": "tsc && concurrently -k -r \"tsc --watch --preserveWatchOutput\" \"wtr --watch\"",
     "storybook": "tsc && npm run analyze -- --exclude dist && concurrently -k -r \"tsc --watch --preserveWatchOutput\" \"wds -c .storybook/server.mjs\"",
     "storybook:build": "tsc && npm run analyze -- --exclude dist && build-storybook",
-    "gen-widget": "cwb gen-widget --propertiesFile uidText.json --webComponentBundle dist/uid-text.bundle.min.js --outputDir uid_widget",
+    "gen-widget": "npm run bundle && cwb gen-widget --propertiesFile uidText.json --webComponentBundle dist/uid-text.bundle.min.js --outputDir uid_widget",
```
Comment (benjaminParisel) [inline components/uid-text/package.json:34]: :ok_hand: au top !

**Review by benjaminParisel — APPROVED** [review]
Some comments

File: README.md [hunk 2]
```diff
@@ -2,3 +2,269 @@
...
+  ```bash
+  $ cd components
+  $ cwb duplicate-widget --srcDir uid-input --destDir my-input
+  ````
+
+---
+### Install its dependencies:
+  ```bash
+  $ cd my-input
+  $ npm install
```
[the remark this answers is not in the document]
Comment (jeromecambon) [reply README.md:36]: I have added a "init" script in package.json that does both install

File: README.md [hunk 3]
```diff
@@ -2,3 +2,269 @@
...
+---
+### Test your component  
+Add the necessary unit tests (in the `test` folder), and run:
+  ```bash
+  $ npm run test
+  ```
+
+---
+### Update the properties json file
+> **Note**: The properties files is an UI Designer specific json file, which defines all the information required to handle the web component in the UI Designer editor (properties, templates, icon, ...).
```
[the remark this answers is not in the document]
Comment (jeromecambon) [reply README.md:63]: Done

File: README.md [hunk 4]
```diff
@@ -2,3 +2,269 @@
...
+
+
+### Generating the UI Designer properties file
+
+> **Note**: The properties files is an UI Designer specific json file, which defines all the information required to handle the web component in the UI Designer editor (properties, templates, icon, ...).
+
+Generate the properties file using the [cwb CLI](https://github.com/bonitasoft/bonita-ui-designer-sdk/blob/main/custom-widget-builder/README.md#main-usage).
+If the generator cannot get any information to generate the properties file, please consider either adding information to
+your components using JSDoc (see above), or generate a properties template file using the CLI.
+
```
[the remark this answers is not in the document]
Comment (jeromecambon) [reply README.md:151]: Done

**Pull request merged**